app/scripts/modules/core/pipeline/config/triggers/git/gitTrigger.html

```diff
@@ -0,0 +1,46 @@
+<ng-form name="cronForm">
+  <div class="form-group">
+    <label class="col-md-3 sm-label-right">
+      Repository Type
+      <help-field key="pipeline.config.git.repotype"></help-field>
+    </label>
+    <div class="col-md-6">
+      <select
+        class="form-control input-sm"
+        ng-options="repoType for repoType in ['stash', 'github']"
```

Comment:
This should be configurable - while Stash makes sense for our needs, I doubt many users outside Netflix are going to want it as an option, much less the default option.